### PR TITLE
[13.0][MIG] purchase_reception_notify

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Standard migration

This module notify on PO chatter the reception of linked moves when they are validated.

CC @ForgeFlow

### CODE REVIEW

File: purchase_reception_notify/__manifest__.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+# Copyright 2019 ForgeFlow S.L.
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl-3.0).
 
 {
     "name": "Purchase Reception Notify",
-    "version": "12.0.1.0.0",
+    "version": "13.0.1.0.0",
     "category": "Purchase Management",
-    "author": "Eficent, " "Odoo Community Association (OCA)",
+    "author": "ForgeFlow, " "Odoo Community Association (OCA)",
```
Comment (LoisRForgeFlow) [inline purchase_reception_notify/__manifest__.py:8]: make it one single String: `"blabla " "another blabla"` ->  `"blabla another blabla"`

File: purchase_reception_notify/tests/test_purchase_reception_notify.py [hunk 2]
```diff
@@ -15,13 +15,13 @@ def setUp(self):
...
         # partners
-        pa_dict = {"name": "Partner 1", "supplier": True}
+        pa_dict = {"name": "Partner 1"}
         self.partner = partner_model.sudo().create(pa_dict)
 
         # Purchase Order Num 1
         po_dict = {"partner_id": self.partner.id}
         self.purchase_order = self.purchase_order_model.create(po_dict)
-        uom_id = self.product_uom_model.search([("name", "=", "Unit(s)")])[0].id
+        uom_id = self.product_uom_model.search([("name", "=", "Units")])[0].id
```
Comment (LoisRForgeFlow) [inline purchase_reception_notify/tests/test_purchase_reception_notify.py]: isn't it better to use the external id: `self.env.ref('uom.product_uom_unit')`?

**Review by LoisRForgeFlow — COMMENTED** [review]
Tested on runbot and works fine, 2 minor comments inline:

File: purchase_reception_notify/tests/test_purchase_reception_notify.py (hunk 2 above)
Comment (HviorForgeFlow) [reply purchase_reception_notify/tests/test_purchase_reception_notify.py]: Done! Thanks!

**Review by AdriaGForgeFlow — APPROVED** [review]
Functional and code review :+1:

**Review by LoisRForgeFlow — APPROVED** [review]
:+1:

**Pull request merged**